src/Amazon.Lambda.Tools/TemplateProcessor/TemplateProcessorManager.cs

```diff
@@ -142,12 +142,18 @@ private async Task<UpdateResourceResults> ProcessUpdatableResourceAsync(string t
             UpdateResourceResults results;
             var localPath = field.GetLocalPath();
 
-            if (!Path.IsPathRooted(localPath))
+            if (!field.IsImageUri && !Path.IsPathRooted(localPath))
```

Comment:
We don't know if there were users that were relying on the .NET CLI building images with just the `ImageUri` property set. 

We can look to see if the value for ImageUri contains `.dkr.ecr.` and the first token is a numeric. If so then we can assume it is an ECR image and use that otherwise run the local build.

Reply:
Added extra check in `IsCode` and `IsImageUri` properties in `UpdatableResource.cs`. Please review.